components/atom/progressBar/src/index.scss

```diff
@@ -0,0 +1,42 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$bg-progress-bar: $c-gray-lightest;
```

Comment:
Should these be default values in order to be custom or are not customizable any of them?

Reply:
I don't see anything in the specification regarding this but I understand that these colors should be customizable

Reply:
Ok! We could ask UX guys in order to certify this, but if the colors are customizable then we should use `!default` in these variables.